{
	"id": "ELSA-2023-12348",
	"advisories": [
		{
			"content": {
				"id": "ELSA-2023-12348",
				"title": "ELSA-2023-12348:  python3-setuptools security update (IMPORTANT)",
				"description": "[39.2.0-10.0.1]\n- Back port fix for CVE-2022-40897 [Orabug: 35034581]",
				"severity": [
					{
						"type": "vendor",
						"source": "linux.oracle.com/security",
						"vendor": "IMPORTANT"
					}
				],
				"references": [
					{
						"source": "linux.oracle.com/security",
						"url": "https://linux.oracle.com/cve/CVE-2022-40897.html"
					},
					{
						"source": "linux.oracle.com/security",
						"url": "https://linux.oracle.com/errata/ELSA-2023-12348.html"
					}
				],
				"published": "2023-05-23T00:00:00Z"
			},
			"ecosystems": [
				"oracle:7"
			]
		}
	],
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2022-40897",
				"references": [
					{
						"source": "linux.oracle.com/security",
						"url": "https://linux.oracle.com/cve/CVE-2022-40897.html"
					}
				]
			},
			"ecosystems": [
				"oracle:7"
			]
		}
	],
	"detection": [
		{
			"ecosystem": "oracle:7",
			"criteria": {
				"operator": "OR",
				"criterions": [
					{
						"vulnerable": true,
						"package": {
							"name": "python3-setuptools",
							"architectures": [
								"aarch64",
								"x86_64"
							]
						},
						"affected": {
							"type": "rpm",
							"range": [
								{
									"lt": "0:39.2.0-10.0.1.el7"
								}
							],
							"fixed": [
								"0:39.2.0-10.0.1.el7"
							]
						}
					}
				]
			}
		}
	],
	"data_source": "oracle"
}
